packages/web-components/@sfx/search-box/README.md

```diff
@@ -6,19 +6,19 @@ The component accepts text input and dispatches events based on input.
 
 ### Events
 
-#### `sfx::searchbox_clear_click`
+#### `sfx::searchbox_cleared`
```

Comment:
We should be consistent with tenses. This should be `searchbox_clear`, or the event below should be `searchbox_clicked`.

Reply:
Will do. I think we went with "cleared" early on because it was a reaction event, but there are no DOM events in the past tense.